src/Simulation/Simulators/QCTraceSimulator/QCTraceSimulator.cs

```diff
@@ -98,6 +98,14 @@ public class QCTraceSimulatorConfiguration
         /// </summary>
         public bool UseWidthCounter = false;
 
+        /// <summary>
+        /// Constrols if depth or width optimization is favored.
+        /// If set to <c>true</c>, resulting circuit is optimized for depth by discouraging qubit reuse.
+        /// If set to <c>false</c>, resulting circuit is optimized for width be encouraging qubit reuse.
+        /// Optimization is only limited to reuse of qubits after they are released by user code.
+        /// </summary>
+        public bool OptimizeDepth;
```

Comment:
Should this have a default value, similar to other fields in the class?

Reply:
A result of playing with different settings. Thanks for catching. Fixed.